src/allmydata/web/status.py

```diff
@@ -1551,6 +1552,31 @@ def render_JSON(self, req):
         req.setHeader("content-type", "text/plain")
         return json.dumps(stats, indent=1) + "\n"
 
+    @render_exception
+    def render_OPENMETRICS(self, req):
```

Comment:
A docstring here, maybe including a reference about openmetric, would be great.

Reply:
Hope you like it :)